src/lib/menu/menu-trigger.ts

```diff
@@ -104,6 +107,14 @@ export class MdMenuTrigger implements AfterViewInit, OnDestroy {
       this._createOverlay();
       this._overlayRef.attach(this._portal);
       this._subscribeToBackdrop();
+
+      this._globalKeydownHandler = this._renderer.listen('document', 'keydown',
```

Comment:
I believe it was just for the visual treatment. I don't think there's any reason not to focus the options in itself.